### PR TITLE
[FR] rust translation chap9 end

### PR DESCRIPTION
**Pull request by mgemard**

### CODE REVIEW

File: lessons.yaml [hunk 1]
```diff
@@ -7006,6 +7018,16 @@ pages:
...
 
         Ferris says, "Be a good rustacean and help a fellow crab out with a good prelude!"
+    fr:
+      title:  Crée tes propres modules prelude
+      content_markdown: |
+        Il est courant qu'une bibliothèque ait son propre module prelude. Cela
+        permet d'importer toutes les structures de données nécessaires à
+        l'utilisation de la bibliothèque. Ces structures de données ne seront
+        pas automatiquement disponible (il faudra utiliser
+        `use my_library::prelude::*`) mais c'est une bonne convention à respecté.
```
Comment (VincentFoulon80) [inline lessons.yaml:7028]:
```suggestion
        Il est courant qu'une bibliothèque ait son propre module prelude. Cela
        permet d'importer toutes les structures de données nécessaires à
        l'utilisation de la bibliothèque. Ces structures de données ne seront
        pas automatiquement disponible (il faudra utiliser
        `use my_library::prelude::*`) mais c'est une bonne convention à respecter.
```

File: lessons.yaml [hunk 2]
```diff
@@ -7014,6 +7036,17 @@ pages:
...
         Resources:
         - [Guidelines For Writing Rust APIs](https://rust-lang.github.io/api-guidelines/)
+    fr:
+      title: Chapter 9 - Conclusion
+      content_markdown: |
+        Tu dois maintenant en connaître beaucoup sur la création de bibliothèques
+        et d'applications Rust. Je ne te demande pas de tout retenir car je suis
+        sûr que, à mesure que ta bibliothèque ou ton application s'agrandira et sera
+        utilisé par d'autres développeurs, tu sera capable de trouver une
+        solution à chaque problême que tu rencontrera.
```
Comment (VincentFoulon80) [inline lessons.yaml:7046]:
```suggestion
        Tu dois maintenant en connaître beaucoup sur la création de bibliothèques
        et d'applications Rust. Je ne te demande pas de tout retenir car je suis
        sûr que, à mesure que ta bibliothèque ou ton application s'agrandira et sera
        utilisé par d'autres développeurs, tu seras capable de trouver une
        solution à chaque problème que tu rencontrera.
```

**Comment by richardanaya** [comment]
Awesome!

**Comment by richardanaya** [comment]
Published

**Pull request merged**